tests/TitleTest.php

```diff
@@ -805,6 +805,51 @@ public function testMpaa()
         }
     }
 
+    public function testMpaa_should_return_the_last_rating_for_each_country_when_no_bool_param()
```

Comment:
You can skip this one as long as you use the same movie in testMpaa and testMpaa_multiple_ratings.